### PR TITLE
(apiExplorer): add override to align vertically to top

### PR DESCRIPTION
**Pull request by jcaianirh**
Fix issue: https://github.com/openshift/console/issues/2386

Screen Shot 2019-08-17 at 6 11 33 PM

### CODE REVIEW

File: frontend/public/style/_overrides.scss [hunk 1]
```diff
@@ -397,6 +397,10 @@ kbd {
   vertical-align: top;
 }
 
+.pf-c-table.pf-m-compact tr > td {
```
Comment (spadgett) [inline frontend/public/style/_overrides.scss]: Should we generalize the `.pf-c-table.pf-c-virtualized` rule above and update the comment instead?

At one point, I believe this caused problems for mobile display with table `gridBreakPoint`. We're not using that currently, however, so maybe it's a non-issue.

cc @rhamilto

Comment (rhamilto) [reply frontend/public/style/_overrides.scss]: +1.  Since the declarations match, it'd be great to combine the rules in to a single one.

Comment (rhamilto) [reply frontend/public/style/_overrides.scss]:
```css
.pf-c-table.pf-m-compact,
// override .co-m-table-grid [class*='col-'], .co-m-table-grid .row
.pf-c-table.pf-c-virtualized {
  tr > td {
    vertical-align: top;
  }
}
```

**Pull request merged**